dev-qt/qtwebkit/qtwebkit-5.212.0_pre20180120.ebuild

```diff
@@ -0,0 +1,145 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+COMMIT=72cfbd7664f21fcc0e62b869a6b01bf73eb5e7da
+CMAKE_MAKEFILE_GENERATOR="ninja"
+PYTHON_COMPAT=( python2_7 )
+QT_MIN_VER="5.9.1:5"
+USE_RUBY="ruby23 ruby24 ruby25"
+inherit check-reqs cmake-utils flag-o-matic python-any-r1 qmake-utils ruby-single toolchain-funcs
+
+DESCRIPTION="WebKit rendering library for the Qt5 framework (deprecated)"
+HOMEPAGE="https://www.qt.io/"
+SRC_URI="http://code.qt.io/cgit/qt/${PN}.git/snapshot/${COMMIT}.tar.gz -> ${P}.tar.gz"
+
+if [[ ${PV} != *9999* ]]; then
+	KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~x86"
+fi
+
+SLOT=5/5.212
+LICENSE="BSD LGPL-2+"
+IUSE="+geolocation gles2 +gstreamer +hyphen +jit multimedia nsplugin opengl orientation +printsupport qml webp X"
```

Comment:
I'm not sure if I'd enable `geolocation` by default... qtwebengine and previous versions of qtwebkit don't, and we've traditionally been rather minimalistic in this area.